### PR TITLE
Feature Flag for health of server dashboard

### PR DESCRIPTION
**Pull request by BenCookie95**
#### Summary
New feature flag for health of server dashboard

#### Ticket Link
https://community.mattermost.com/boards/workspace/7dqusmf8stn3dens5ckhyatbdo/boattb15z9fr4m85etwrdekkduh/v37ny5c5eafnk3dypexwzagip7a/ckwt5dond5b89ignwamfuk18e5y

#### Release Note
```release-note
Feature flag for health of server dashboard
```

### CODE REVIEW

File: model/feature_flags.go [hunk 1]
```diff
@@ -72,6 +72,9 @@ type FeatureFlags struct {
 	BoardsDataRetention bool
 
 	NormalizeLdapDNs bool
+
+	// Enable health of server dashboard
+	HealthOfServerDashboard bool
```
Comment (michelengelen) [inline model/feature_flags.go:77]: It is now "WorkspaceOptimizationDashboard" ... could you change that to the new name please? 👍🏻🙇🏼‍♂️

**Review by michelengelen — APPROVED** [review]
Approving, since the only thing is just renaming the Flag

**Review by mkraft — APPROVED** [review]

**Comment by mkraft** [comment]
@furqanmlk Please be aware I deferred QA on this feature flag addition.

**Comment by cwarnermm** [comment]
Documented via https://github.com/mattermost/docs/pull/5464

**Pull request merged**